### PR TITLE
Addresses PKS-156: Harbor "remote error: tls: handshake failure" when connecting to S3

### PR DESCRIPTION
**Pull request by lparis**
Under review.

### CODE REVIEW

File: docs-content/installing.html.md.erb [hunk 1]
```diff
@@ -158,6 +158,8 @@ If you choose **AWS S3**, configure the following settings:
 
     <img src="images/aws.png" alt="AWS S3 configuration for Harbor" width="725">
 
+<p class="note"><strong>Note:</strong> When using Harbor with an S3 compatible object store, the object store must be configured with a TLS cipher suite supported by Harbor. If the S3 bucket is not configured with a compatible cipher suite, when performing a `docker push` command to or from the Harbor registry, you receive the following: "remote error: tls: handshake failure". If you run a tlsscan on the host where the error occurs, you will get a list of supported cipher suites. To address the error, enable any cipher suite in the list that has the prefix `TLS_ECDHE_RSA`.</p>
```
Comment (stonezdj) [inline docs-content/installing.html.md.erb]: the object store must be configured with a TLS cipher suite supported by docker client.

Comment (michmike) [reply docs-content/installing.html.md.erb]: it is important to make the distinction that it is not Harbor that is requires this, but the docker client
The handshake failure happened between the docker client and the S3-compatible storage

@stonezdj do you agree we should make that distinction in the docs?

**Comment by stonezdj** [comment]
Yes, It should be:

When using Harbor with an S3 compatible object store, the object store must be configured with a TLS cipher suite supported by docker client. If the S3 bucket is not configured with a compatible cipher suite, when performing a `docker push` command to the Harbor registry, you receive the following: "remote error: tls: handshake failure". The Harbor will redirect the connection from docker client to S3 compatible object store, It is the issue between the docker client and s3 compatible object store. To address this error, you should know the cipher suites supported by docker client and s3 compatible object store, and guarantee they have a common cipher suite at least.

**Comment by lparis** [comment]
@stonezdj @michmike Changes and distinctions made; inaccuracies corrected. Please review and approve ASAP. Thanks.

File: docs-content/installing.html.md.erb [hunk 2]
```diff
@@ -158,6 +158,8 @@ If you choose **AWS S3**, configure the following settings:
 
     <img src="images/aws.png" alt="AWS S3 configuration for Harbor" width="725">
 
+<p class="note"><strong>Note:</strong> When using Harbor with an S3 compatible object store, the object store must be configured with a TLS cipher suite supported by the Docker client. If the S3 bucket is not configured with a compatible cipher suite, when performing a `docker push` operation, you receive the following: "remote error: tls: handshake failure". If you run a tlsscan on the host where the Docker client is installed, you will get a list of supported cipher suites. To address the error, enable any cipher suite in the list that has the prefix `TLS_ECDHE_RSA`. These cipher suites are hardcoced in the Docker client.</p>
```
Comment (michmike) [inline docs-content/installing.html.md.erb]: we need to be clear that enabling the cipher suite is on the S3 side, correct?

**Comment by lparis** [comment]
Please merge and deploy ASAP. Thanks.

**Comment by animatedmax** [comment]
@lparis This is now published: https://docs.pivotal.io/partners/vmware-harbor/installing.html#aws

**Comment by lparis** [comment]
Thanks a lot for getting this published quickly.

**Pull request merged**